test/unittest/StatisticsBackend/IsMetatrafficTests.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest_aux.hpp>
+#include <gtest/gtest.h>
+
+#include <database/entities.hpp>
+#include <StatisticsBackend.hpp>
+#include <StatisticsBackendData.hpp>
+
+using namespace eprosima::statistics_backend;
+using namespace eprosima::statistics_backend::database;
+
+constexpr const char* PID_DEFAULT = "36000";
+constexpr const char* DATA_TYPE_DEFAULT = "data_type";
+#define QOS_DEFAULT Qos(R"({"qos":"empty"})"_json)
+
+
+#define LOCATOR_DEFAULT_NAME(x) "locator_" + std::to_string(x)
+#define HOST_DEFAULT_NAME(x) "host_" + std::to_string(x)
+#define USER_DEFAULT_NAME(x) "user_" + std::to_string(x)
+#define PROCESS_DEFAULT_NAME(x) "process_" + std::to_string(x)
+#define DOMAIN_DEFAULT_NAME(x) "12" + std::to_string(x)
+#define ALIAS_DEFAULT_NAME(x) "domain_" + std::to_string(x)
+#define TOPIC_DEFAULT_NAME(x) "topic_" + std::to_string(x)
```

Comment:
Use more than one topic to test it is or it is not metatraffic
```suggestion
#define NON_METATRAFFIC_TOPIC_DEFAULT_NAMES(x) {"topic_" + std::to_string(x), "rt_" ...
```

Reply:
:+1: